pkg/actors/actors.go

```diff
@@ -1143,11 +1182,84 @@ func (a *actorsRuntime) doExecuteReminderOrTimer(ctx context.Context, reminder *
 	return nil
 }
 
+func scheduleFromPeriod(period string) (*string, *uint32, error) {
+	if len(period) == 0 {
+		return nil, nil, nil
+	}
+
+	years, months, days, duration, repetition, err := kittime.ParseDuration(period)
+	if err != nil {
+		return nil, nil, fmt.Errorf("unsupported period format: %s", period)
+	}
+
+	if years > 0 || months > 0 || days > 0 {
+		return nil, nil, fmt.Errorf("unsupported period format: %s", period)
+	}
+
+	var repeats *uint32
+	if repetition > 0 {
+		repeats = ptr.Of(uint32(repetition))
+	}
+
+	return ptr.Of("@every " + duration.String()), repeats, nil
+}
+
 func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderRequest) error {
 	if !a.actorsConfig.Config.HostedActorTypes.IsActorTypeHosted(req.ActorType) {
 		return ErrReminderOpActorNotHosted
 	}
 
+	if a.schedulerClients != nil && a.schedulerReminders {
```

Comment:
Please move to a separate function for readability

Reply:
sure thing